spec/models/manageiq/providers/kubernetes/container_manager/scanning/job_spec.rb

```diff
@@ -17,4 +67,125 @@
                               :role        => 'smartstate')).to be true
     end
   end
+
+  context "A single Container Scan Job," do
+    IMAGE_ID = '3629a651e6c11d7435937bdf41da11cf87863c03f2587fa788cf5cbfe8a11b9a'.freeze
+    IMAGE_NAME = 'test'.freeze
```

Comment:
@moolitayer I see you added `freeze`, is it related to this PR?
(It's a good optimization but it will slip in unnoticed within this PR)